templates/includes/pagination.html

```diff
@@ -0,0 +1,15 @@
+<ul class="pagination">
+  {% if page_obj.has_previous %}
+    <li class="page-item">
+      <a href="?page= {{ page_obj.previous_page_number }}" class="page-link">prev</a>
+    </li>
+  {% endif %}
+  <li class="page-item active">
+    <span class="page-link">{{ page_obj.number }} of {{ paginator.num_pages }}</span>
+  </li>
+  {% if page_obj.has_next %}
+    <li class="page-item">
+      <a href="?page= {{ page_obj.next_page_number }}" class="page-link">next</a>
```

Comment:
There is an extra space after the equal sign in the href attribute. It should be `href="?page={{ page_obj.next_page_number }}"` without the space.